master/buildbot/test/unit/test_scripts_cleanupdb.py

```diff
@@ -43,6 +43,19 @@ def mkconfig(**kwargs):
     return config
 
 
+def patch_environ(case, key, value):
```

Comment:
I don't think that being more specific doesn't make the code any clearer, so the code might as well be this generic. Also. there are a number of other places in the test suite that appear to modify the environment that could benefit from this.